README.md

```diff
@@ -154,6 +199,42 @@ produce early errors because of recursive application of the first rule.
 - Let _targetName_ be ? Get(_target_, `"name"`).
 - If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
 - Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
+- Assert: _status_ is never used.
```

Comment:
The assertion is just making something explicit that's otherwise implicit. (As in, long time down the road, no one's filing an ECMAScript bug for "Why isn't this used?")

And I'm returning the function, but I'm fixing the parameter name (which there was already another mistake where an undefined _F_ was in the preceding line).